feat: support string entry (#328)
steelbrain/package-deps: README.md

}
```

> or if only the name of the package is available, you specify the name directly as a string for that entry:

Author: ```suggestion
If only the name of the package is needed, you can specify the name directly as a string instead of an object for that entry:
```
